Cargo.toml

```diff
@@ -14,13 +14,6 @@ homepage = "http://maidsafe.net"
 [dependencies]
 rustc-serialize = "*"
 cbor = "*"
-# sodiumoxide = "0.0.4"
+sodiumoxide = "0.0.4"
```

Comment:
It should work with sodium oxide = "0.0.5" is that version stable by now (i.e. it builds) in sodiumoxide?